onnx/test/parser_test.py

```diff
@@ -218,6 +218,36 @@ def test_parse_node(self):
         self.assertEqual(node.domain, "SomeDomain")
         self.assertEqual(node.op_type, "SomeOp")
 
+    @parameterized.expand(
+        [
+            "not_a_good_float",  # Not a good float value, but shouldn't blow up the parser.
+            "inf",
+            "-inf",
+            "infinity",
+            "-infinity",
+            "-NaN",
+            "nan",
+        ]
+    )
+    def test_parse_various_float_values(self, test_literal):
+        model_text = (
+            """
```

Comment:
```suggestion
            f"""
```

Last batch of minor suggestions. Thanks!